authors/apps/articles/urls.py

```diff
@@ -9,4 +9,6 @@
          name='articles'),
     path('<slug>', views.ArticleDetailApiView.as_view(),
          name='article-details'),
+    path('<slug>/rate/', views.ArticleRatingAPIView.as_view(),
```

Comment:
@missvicki I think you should get rid of the trailing slash to ensure consistency.

Reply:
Noted, thank you

Reply:
This has been worked on